style.css

```diff
@@ -250,3 +236,329 @@ body {
   display: inline-block;
   width: 20px;
 }
+
+/* Recent Works section */
+
+.recent-work {
+  background-color: white-background-color;
+  padding: 80px 0;
+  position: relative;
+}
+
+.about-me {
+  position: relative;
+  top: 50px;
+}
+
+.recent-work-container {
+  width: 100%;
+  max-width: 600px;
+  text-align: center;
+  z-index: 1;
+  align-items: center;
+  margin: auto;
+}
+
+.section-title {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin-bottom: 20px;
+  position: relative;
+}
+
+.recent-work h1 {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  letter-spacing: 0;
+  text-align: center;
+  align-items: center;
+  margin-bottom: 40px;
+  color: headline-color;
+}
+
+.title-container {
+  position: relative;
+}
+
+.section-indicator {
+  display: inline-block;
+  width: 48px;
+  height: 8px;
+  background-color: #36b37e;
+  border-radius: 24px;
+  bottom: -3px;
+  left: 50%;
+  transform: translateX(-50%);
+}
+
+.grid {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(300px, 1fr));
```

Comment:
- [ ] Your display grid property has not been done properly because the mobile version should be responsive until the desktop screen size. Kindly fix it such that all three cards are arranged in a row when on the desktop screen and in column when on the mobile screen.